build-bench-env.sh

```diff
@@ -497,6 +503,23 @@ if test "$setup_lt" = "1"; then
   popd
 fi
 
+if test "$setup_pa" = "1"; then
+  checkout pa $version_pa https://github.com/1c3t3a/partition_alloc_builder.git
+
+  # Setup depot_tools for building
+  git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
```

Comment:
You might want to use `checkout` instead I think, or at least call `git clone` with `--depth=1` or something